db/migrate/20200403165044_allow_official_document_type_to_be_null.rb

```diff
@@ -0,0 +1,9 @@
+class AllowOfficialDocumentTypeToBeNull < ActiveRecord::Migration[6.0]
+  def up
+    change_column :file_attachment_metadata_revisions, :official_document_type, :string, null: true, default: nil
+  end
+
+  def down
+    change_column :file_attachment_metadata_revisions, :official_document_type, :string, null: false, default: "unofficial"
+  end
```

Comment:
Well done for defining the `down` method - we quite often don't bother!

Reply:
It may have forced me 😜